src/lxc/start.c

```diff
@@ -174,6 +174,34 @@ static bool preserve_ns(int ns_fd[LXC_NS_MAX], int clone_flags, pid_t pid)
 	return false;
 }
 
+static int set_oom_score_adj(int oom_score_adj, pid_t pid)
+{
+	int ret;
+	FILE *f;
+/* 5 /proc + 21 /int_as_str + 14 /oom_score_adj + 1 \0 */
+#define OOM_PATH_LEN 45
```

Comment:
Nit: Please make this `__OOM_PATH_LEN`. I usually make `#defines` in headers without `__` prefixes and `#defines` local to a file with `__`.